DoSomethingArchive/legacy-website
# Makes Static content translatable

#### What's this PR do?

Makes Static content fields translatable:

Translated:
- `title`
- `field_intro_title`
- `field_cta_link`
- `field_additional_text_title`
- `field_additional_text`

Already has been translated:
- `field_subtitle`
- `field_call_to_action`

To ignore:
- `field_blocks`
- `field_hero_image`
- `field_video`

Not translated, because of an error:
- `field_gallery`, fails with error, possibly related to #5509

Not translated, because defined in other content types:
- `field_intro`
- `field_intro_image`
- `field_partners`
#### How should this be manually tested?
- Enable
- Execute `ds-global make-static-content-translatable`
- Check Static Content; nodes, update, translate, etc...
#### What are the relevant tickets?

Fixes #4992
